modules/nodes-pods-user-namespaces-configuring.adoc

```diff
@@ -0,0 +1,329 @@
+          type null_device_t;
+          type zero_device_t;
+          type urandom_device_t;
+          type random_device_t;
+          type container_file_t;
+          type container_engine_t;
+          type devtty_t;
+          type setfiles_t;
+          class chr_file { mounton read setattr write };
+          class sock_file mounton;
+        }
+
+        #============= container_engine_t ==============
+        allow container_engine_t container_file_t:sock_file mounton;
+        allow container_engine_t devtty_t:chr_file mounton;
+        allow container_engine_t null_device_t:chr_file { mounton setattr };
+        allow container_engine_t random_device_t:chr_file mounton;
+        allow container_engine_t urandom_device_t:chr_file mounton;
+        allow container_engine_t zero_device_t:chr_file mounton;
+systemd:
+  units:
+  - contents: |
+      [Unit]
+      Description=Modify SeLinux Type container_engine_t
+      DefaultDependencies=no
+      After=kubelet.service
+
+      [Service]
+      Type=oneshot
+      RemainAfterExit=yes
+      ExecStart=bash -c "/bin/checkmodule -M -m -o /tmp/nested_podman.mod /etc/nested_podman/nested_podman.te && /bin/semodule_package -o /tmp/nested_podman.pp -m /tmp/nested_podman.mod && /sbin/semodule -i /tmp/nested_podman.pp"
+      TimeoutSec=0
+
+      [Install]
+      WantedBy=multi-user.target
+    enabled: true
+    name: systemd-nested-podman-selinux.service
+----
++
+This machine config enables pods to access the `/dev/fuse` and `/dev/net/tun` devices on the host node.  It also enables some kernel modules required for rootless networking, and modifies the `container_engine_t` Selinux type to allow the file access needed for networking with the `/dev/net/tun` device.
+
+.. Use Butane to generate a `MachineConfig` object file, `40-worker-custom-journald.yaml`, containing the configuration to be delivered to the nodes:
++
+[source,terminal]
+----
+$ butane nested-podman-worker.bu -o nested-podman-worker.yaml
+----
+
+.. Apply the machine config. For example:
++
+[source,terminal]
+----
+$ oc apply -f nested-podman-worker.yaml
+----
++
+The controller detects the new `MachineConfig` object and generates a new `rendered-worker-<hash>` version.
+
+. Enable the pods in the individual user namespace to access the `/dev/fuse` and `/dev/net/tun` devices on the host node.  
+
+.. Create a machine config similar to the following example:
++
+[source,terminal]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 99-worker-allowed-devices
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - path: /etc/crio/crio.conf.d/99-allowed-devices.conf
+        mode: 0640
+        overwrite: true
+        contents:
+          source: data:text/plain;charset=utf-8;base64,W2NyaW8ucnVudGltZV0KYWxsb3dlZF9kZXZpY2VzID0gWyIvZGV2L2Z1c2UiLCAiL2Rldi9uZXQvdHVuIl0K <1>
```

Comment:
duplicated with https://github.com/openshift/openshift-docs/pull/81592/files#r1765363919, and we updated the config tobe available in 4.17.0

Reply:
@haircommander Not sure what you mean here.

Reply:
you double added the allowed-devices.conf, and also we need neither because of an update to cri-o which landed for 4.17.0